modules/iteratorToArray.js

```diff
@@ -0,0 +1,9 @@
+export default function iteratorToArray(iterator) {
```

Comment:
Like with `isIterable`, you should either add a descriptive comment and list this function in `modules/index.js` if you intended this as a public function, or start the module name with `_` otherwise. However, since this function seems to be purely an implementation detail of `_.toArray`, it should probably be internal, and maybe it should even be defined inside `modules/toArray.js`.